projects/Generic/linux/linux.x86_64.conf

```diff
@@ -3147,7 +3147,7 @@ CONFIG_SND_DYNAMIC_MINORS=y
 CONFIG_SND_MAX_CARDS=32
 # CONFIG_SND_SUPPORT_OLD_API is not set
 CONFIG_SND_PROC_FS=y
-# CONFIG_SND_VERBOSE_PROCFS is not set
+CONFIG_SND_VERBOSE_PROCFS
```

Comment:
```
+CONFIG_SND_VERBOSE_PROCFS
```
Shouldn't this be `=y` or something?

And what about other platforms (eg. RPi etc.) - is this option relevant there too? Aren't we still maintaining Nvidia_Legacy for the time being?